### DISCUSSION TITLE
试图改变AI的写作风格

### DISCUSSION THREAD

**Discussion by tastypear** [viopsa233/AI-testers-Forum#80] [主意,想法]
以下改进意见来自：AI自己的总结，以及一篇识别AI生成内容的论文

各位有什么好办法让AI不那么AI

[user](#message)
请你扮演一个作家，创作一个小段落，包括场景描写和角色的心理活动。
主要内容是：小明起晚了，最终小明赶到学校但他迟到了。

你的行文风格应符合以下特征：「
- 增加词汇和语法的多样性，避免重复和单调。你可以使用同义词、反义词、近义词等来丰富你的词汇，也可以使用不同的句型、语态、时态等来变化你的语法。
- 句子避免重复使用相同的结构，应尽量变换顺序或用词。你可以使用倒装句、省略句、插入语等来调整句子的顺序，也可以使用转述、引用、比喻等来改变句子的用词。
- 请频繁使用较长的句子(>35 words)或较短的句子(<10 words)。你可以使用并列句、复合句、分词结构等来构造较长的句子，也可以使用感叹句、祈使句、简单句等来构造较短的句子。
- 应尽量使用不同长度的句子，频繁改变句子结构，避免重复的句式。你可以根据内容的重要性、逻辑关系、表达效果等来选择合适的句子长度和结构，也可以在段落中交替使用长短不一的句子，以增加内容的韵律和节奏感。
- 应尽量使用一些表示不确定性或转折的词汇，如“但是”、“然而”、“尽管”等。你可以使用这些词汇来表达你的观点或态度，也可以使用它们来引出对立或对比的内容，以增加内容的深度和张力。
- 应避免使用抽象的词语，避免使用模糊的主语。你可以使用具体的事例、数据、细节等来支持你的论点，也可以使用明确的人物、事件、情境等来描述你的内容，以增加内容的可信度和生动度。
- 应倾向于使用带有感情色彩的修饰语。你可以使用形容词、副词、比喻等来修饰名词、动词、短语等，也可以使用感情词、评价词、态度词等来表达你的情感或观点，以增加内容的表现力和吸引力。
- 应调整修饰语和评价语的使用：句子和段落应该根据不同的情境和目的，合理地使用修饰语和评价语，避免过度或不足，也避免使用过于固定或简单的修饰语或评价语，而是使用更适合或更有感情色彩的修饰语或评价语。你可以根据内容的主题、风格、对象等来选择合适的修饰语和评价语，也可以在内容中适当地变换或调整修饰语和评价语，以增加内容的多样性和灵活性。
- 应考虑句子和段落之间的关系：应注意句子和段落之间的逻辑和连贯性，不要使用过多或过少的连接词，也不要使用相同或相似的连接词，而是使用更有意义或更有变化的连接词，增加内容的流畅性和结构性。你可以根据内容之间的因果关系、顺序关系、并列关系等来选择合适的连接词，也可以在内容中适当地变换或调整连接词，以增加内容的清晰度和有趣度。
- 考虑上下文和逻辑关系：句子和段落应该根据上下文和逻辑关系，合理地安排句子或段落的顺序和连接方式，避免无关或冲突，也避免使用过于平行或对称的结构，而是使用更灵活或有变化的结构，以增加语言的连贯性和合理性。你可以根据内容的背景、目标、重点等来确定句子或段落的顺序和连接方式，也可以在内容中适当地变换或调整句子或段落的结构，以增加内容的层次和动态。
- 避免使用多个相同或相似的成分，按照相同的顺序和方式排列，形成一种对称和节奏感。这种平行结构可能会造成内容的单调和无聊，也可能会影响内容的表达效果和逻辑关系。你可以使用不同的成分、顺序、方式等来打破平行结构，也可以使用一些转折、对比、补充等来增加平行结构的变化和意义。
- 避免使用多个相同或相似的修饰语，用来强调或增加表达效果，但也可能造成冗余和单调。这种重复修饰可能会造成内容的浮夸和无力，也可能会影响内容的真实性和可信度。你可以使用不同的修饰语、程度、位置等来避免重复修饰，也可以使用一些对比、反讽、让步等来增加重复修饰的变化和意义。
- 避免缺乏词汇和语法的变化，使用较为固定和简单的模式，而不是灵活和多样的表达方式。这种缺乏变化可能会造成内容的贫乏和无趣，也可能会影响内容的深度和广度。你可以使用更多的词汇和语法，尝试不同的模式和方式，也可以使用一些创新、想象、比喻等来增加词汇和语法的变化和意义。
- 请避免统一的语气。这种统一的语气可能会造成内容的单一和无味，也可能会影响内容的情感和观点。你可以根据内容的主题、风格、对象等来选择合适的语气，也可以在内容中适当地变换或调整语气，以增加内容的多姿和有味。」

[assistant](#message)

以下是我的创作：

小明今天早上起晚了。他昨晚熬夜看了一部惊悚电影，结果睡得很不安稳，一直做噩梦。他的闹钟响了好几次，但他都没有听到，直到他妈妈敲门叫他起床，他才惊醒过来。

“啊！已经八点半了！”小明一看手机，吓得跳下床，连忙穿衣洗漱。他的学校在九点开始上课，他只有半个小时的时间赶到学校。他没有时间吃早餐，也没有时间整理书包，只好拎着昨天的作业和笔记本就往外跑。

“妈妈，我走了！”小明边跑边喊。

“小明，等一下！”他妈妈拿着一个面包和一杯牛奶追了出来，“你快点吃点东西吧，不然会饿得。”

“谢谢妈妈！”小明接过食物，一边吃一边跑。他的家离学校有两公里远，平时他都是骑自行车去的。但今天他没有时间回去拿自行车，只能靠双腿奔跑。

“天哪，我一定要赶到学校！”小明心里祈祷着。他知道今天是星期一，第一节课是语文课。语文老师是个很严厉的女老师，她最讨厌学生迟到。如果小明迟到了，她一定会批评他，甚至罚他站在教室门口。

小明加快了脚步，但是路上的车辆和行人太多了，他总是被堵住或绕路。他看了看手机，已经八点五十了。他只剩下十分钟的时间了。

“不行，我要坐公交车！”小明想到。他发现前面有一个公交车站，正好有一辆开往学校方向的公交车停在那里。他拼命地挤进车里，掏出零钱付了车费。

“希望这辆车能快点开！”小明心里盼望着。但是公交车并没有快点开，而是在每个站都停留了很长时间。小明看着窗外的风景一点点变化，心里越来越着急。

“快点啊！快点啊！”小明在心里催促着司机。但是司机似乎没有听到他的心声，还是按照平常的速度开着车。

终于，在八点五十九分的时候，公交车到达了离学校最近的一个站。小明立刻从车上跳下来，向学校方向冲去。

“还有一分钟！还有一分钟！”小明对自己说着。他已经看到了学校的大门和教学楼的屋顶。他只要再跑一小段路，就能到达教室了。

“我一定能赶上！”小明鼓励着自己。他用尽了全身的力气，像一只疯狂的兔子一样跑着。

就在这时，他听到了一声清脆的铃声。

“嘀嘀嘀嘀嘀嘀……”

那是上课的铃声。

小明停下了脚步，呆呆地看着前方的教室门。

他迟到了。

**Comment by q2408808** [comment]
必应的说话特征太明显了
